tests/integration_tests/functional/test_balloon.py

```diff
@@ -700,3 +706,90 @@ def _test_snapshot_compatibility(context):
     )
 
     microvm.kill()
+
+
+@pytest.mark.skipif(
```

Comment:
Why?

Reply:
Forgot to remove the annotation. It's gone now.